server/src/main.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+# Requires:  CherryPy 18.0.1  (www.cherrypy.org)
```

Comment:
Probably don't need these comments, better to update the README/wiki with the libraries we are using.